docker-compose.yml

```diff
@@ -38,4 +37,4 @@ services:
     dns:
       - 127.0.0.1
     labels:
-      - triton.cns.services=mysql-consul
+      - triton.cns.services=mc
```

Comment:
No, I agree it's a terrible name. 😀 

But unfortunately the records that Triton CNS names create are "too long" despite being within spec and this causes the application to be unable to resolve the CNS name for Consul. (And despite having worked at one point in time.)

I've been unable to track down the problem but it's been kicked around between CNS, the Python client, and/or Google's DNS servers which are the search domain for Triton-deployed applications. I've been unable to figure it out but this hacks around that until such time as we can use the Triton search domain.

ref:
https://devhub.joyent.com/jira/browse/DOCKER-898
https://devhub.joyent.com/jira/browse/OPS-2555